hash/m20.xml

```diff
@@ -31,126 +33,164 @@ license:CC0-1.0
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="pcos41a">
+	<software name="pcos30ea" cloneof="pcos30e">
+		<description>Olivetti PCOS 3.0e (Copy 1)</description>
+		<year>1983</year>
+		<publisher>Olivetti</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="286720">
+				<rom name="pcos30e.td0" size="286720" crc="c6de0eec" sha1="2e1eb3d152a92ed1262899208ef298179e1b61b3"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcos30eb" cloneof="pcos30e">
+		<description>Olivetti PCOS 3.0e (Copy 2)</description>
+		<year>1983</year>
+		<publisher>Olivetti</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="286720">
+				<rom name="pcos30e.imd" size="286720" crc="383a2131" sha1="62698a393328dc3ac1cdd5d72360623f741e54e1"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcos41a" supported="partial"><!-- Missing FM track0 (all) -->
 		<description>Olivetti PCOS 4.1a</description>
-		<year>19??</year>
+		<year>1984</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pcos_41a.img" size="286720" crc="f2fa9799" sha1="6d01e7ecc91211e588b4af3c77b972b20deee63b"/>
+				<rom name="pcos_41a.img" size="286720" crc="f2fa9799" sha1="6d01e7ecc91211e588b4af3c77b972b20deee63b" status="baddump"/>
 			</dataarea>
 		</part>
 		<part name="flop2" interface="floppy_5_25">
 			<feature name="part_id" value="Tools" />
 			<dataarea name="flop" size="286720">
-				<rom name="pcos_41a_tools.img" size="286720" crc="24936905" sha1="d27a2cae62fd62f07dce0e3df397d2dea4c1253b"/>
+				<rom name="pcos_41a_tools.img" size="286720" crc="24936905" sha1="d27a2cae62fd62f07dce0e3df397d2dea4c1253b" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="cpm8k">
+	<software name="cpm8k" supported="no"><!-- Missing FM track0, boot with -ramsize 512k -->
 		<description>CP/M 8000</description>
 		<year>19??</year>
 		<publisher>&lt;unknown&gt;</publisher>
+		<info name="usage" value="Launch with: -ramsize 512k" />
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="cpm8k.img" size="286720" crc="e0f8dfe3" sha1="9d6a17af5ef2775ad985bface454926ae3f2a322"/>
+				<rom name="cpm8k.img" size="286720" crc="e0f8dfe3" sha1="9d6a17af5ef2775ad985bface454926ae3f2a322" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="msdos20" supported="no">
+	<software name="msdos20">
 	<!-- "This is a standard 360k MS-DOS disk image, 512 bytes/sector, 9 sectors, 2 sides, 40 tracks." -->
 	<!-- Use with BIOS 2.0 and the 8086 APB -->
-		<description>MS-DOS 2.0</description>
-		<year>19??</year>
+		<description>Olivetti MS-DOS 2.0</description>
+		<year>1983</year>
 		<publisher>Microsoft</publisher>
+		<info name="usage" value="Launch with: -bios 1" />
```

Comment:
Please use the mnemonic BIOS name (`m20-20d` in this case I believe).  The numbers are too unstable.